Update RStudios to R-IDE (#663)
seqeralabs/docs: platform-cloud/docs/getting-started/quickstart-demo/studios.md

This guide provides an introduction to Studios using a demo Studio in the Community Showcase workspace. See [Studios](../../studios/overview) to learn how to create Studios in your own workspace. 
:::

> Interactive analysis of pipeline results is often performed in platforms like Jupyter Notebook or R-IDE. Setting up the infrastructure for these platforms, including accessing pipeline data and the necessary bioinformatics packages, can be complex and time-consuming.

Author: ```suggestion
Interactive analysis of pipeline results is often performed in platforms like Jupyter Notebook or [R-IDE](https://github.com/seqeralabs/r-ide). Setting up the infrastructure for these platforms, including accessing pipeline data and the necessary bioinformatics packages, can be complex and time-consuming.
```

> Interactive analysis of pipeline results is often performed in platforms like Jupyter Notebook or [R-IDE](https://github.com/seqeralabs/r-ide). Setting up the infrastructure for these platforms, including accessing pipeline data and the necessary bioinformatics packages, can be complex and time-consuming.

Author: ```suggestion
Interactive analysis of pipeline results is often performed in platforms like Jupyter Notebook or an [R-IDE](https://github.com/seqeralabs/r-ide). Setting up the infrastructure for these platforms, including accessing pipeline data and the necessary bioinformatics packages, can be complex and time-consuming.
```

### Open the RNAseq analysis Studio 

> Select the *rnaseq_to_differentialabundance* Studio. This Studio consists of an R-IDE environment that uses an existing compute environment available in the Community Showcase workspace. The Studio also contains mounted data generated from the *nf-core/rnaseq* and subsequent *nf-core/differentialabundance* pipeline runs, directly from AWS S3. 

Reviewer: @justinegeffen Notebook or environment?

### Connect to the Studio

> This Studio will start an R-IDE which already contains the necessary R packages for deploying an RShiny application to interact with various visualizations of the RNAseq data. The Studio also contains an R Markdown document with the commands in place to generate the application.

Author: ```suggestion
This Studio will start an R-IDE which already contains the necessary R packages for deploying a web app to interact with various visualizations of the RNAseq data. The Studio also contains an R Markdown document with the commands in place to generate the application.
```

> This Studio will start an R-IDE environment which already contains the necessary R packages for deploying an RShiny application to interact with various visualizations of the RNAseq data. The Studio also contains an R Markdown document with the commands in place to generate the application.

Reviewer: @justinegeffen Notebook or environment?

> This Studio will start an R-IDE environment which already contains the necessary R packages for deploying an RShiny application to interact with various visualizations of the RNAseq data. The Studio also contains an R Markdown document with the commands in place to generate the application.

Author: ```suggestion
This Studio will start an R-IDE which already contains the necessary R packages for deploying an RShiny application to interact with various visualizations of the RNAseq data. The Studio also contains an R Markdown document with the commands in place to generate the application.
```
